Documentation/ceph-fs-subvolumegroup.md

```diff
@@ -35,7 +35,11 @@ If any setting is unspecified, a suitable default will be used automatically.
 
 ### CephFilesystemSubVolumeGroup metadata
 
-- `name`: The name that will be used for the Ceph Filesystem subvolume group.
+* `name`: The name that will be used for the Ceph Filesystem subvolume group.
+* `external`:
+  * `enable`: if `true` the cluster is external and we want to take advantage of the subvolumegroup
```

Comment:
"enabled" is the pattern I believe we generally use instead of the verb
```suggestion
  * `enabled`: if `true` the cluster is external and we want to take advantage of the subvolumegroup
```

Reply:
Reading through the PR, I wonder if it would make more sense to have the field be `isExternal` as a boolean instead of having the two-level external.enabled? It's not really enabled or not, it's different behavior based on external or not. There is also no other external configuration, right?

Reply:
I just re-used the existing spec but you know what now that I think more of it. I could probably check if the CephCluster is external and do the exact same thing. With that, we can avoid the API change in `CephFilesystemSubVolumeGroupSpec`. How that sounds?

Reply:
Perfect, makes sense to check the cluster CR